resume.py

```diff
@@ -60,11 +48,16 @@ def exists(email):
 @ensure_logged_in_user()
 def resume(event, ctx, user):
     try:
-        return {"statusCode": 200, "body": {
-            "upload": presign("put_object", event, ctx, user),
-            "download": presign("get_object", event, ctx, user),
-            "exists": exists(event["email"])
-        }}
+        return add_cors_headers({
+            "statusCode": 200, "body": {
+                "upload": presign("put_object", event, ctx, user),
+                "download": presign("get_object", event, ctx, user),
+                "exists": exists(event["email"])
+            }
+        })
     except ClientError as e:
-        return {"statusCode": 500, "body": "failed to connect to s3" + str(e)}
+        return add_cors_headers({
```

Comment:
Sorta weird that you have to do this... `ensure_schema` should do this for you...

Reply:
you are correct. im thinking about opening an issue to refactor other places like this. like in authorize. also potentially using it as a decorator instead